ckan/lib/search/query.py

```diff
@@ -470,7 +470,7 @@ def _check_query_parser(param: str, value: Any):
                     raise SearchQueryError('Invalid "sort" parameter')
 
                 if "Failed to connect to server" in e.args[0]:
-                    raise SolrConnectionError("Connection Error", message=e.args[0])
```

Comment:
Can we get the details logged with a `log.warning(e.args[0])`? This might help maintainers debug the actual issue

Reply:
Changes were made so that 
**Site Visitors** sees `"Solr returned an error while searching."`, 
**Site Maintainers** sees `"Connection Error: Failed to connect to Solr server."`